src/theme/components.js

```diff
@@ -66,6 +74,29 @@ const components = {
         '&.filter-status .MuiAlert-icon': {
           paddingTop: '17px',
         },
+        '&.paper-light': {
+          ...DropDownMenuRaw.Light,
+
+          '.dropdown-menu': {
+            ...DropDownItemRaw.Light,
+            color: red,
+          },
+        },
+        '&.paper-dark': {
+          ...DropDownMenuRaw.Dark,
+          $nest: {
+            '.MuiList-root': {
+              $nest: {
+                '.MuiMenuItem-root': {
+                  ...DropDownItemRaw.Dark,
+                },
+                '.Mui-selected': {
+                  backgroundColor: Colors.SecondaryActiveDark,
+                },
+              },
+            },
+          },
```

Comment:
I'm not super fan of this, but I'm OK landing this because it's still better than what we had before. I'm concerned that this gives quite a big specificity because of the nested class names.

For example I wonder if the styles on `MuiList` or `MuiMenuItem` could be used more globally inside of nested inside `MuiPaper`.

Reply:
"For example I wonder if the styles on MuiList or MuiMenuItem could be used more globally inside of nested inside MuiPaper."

I'm not sure I understand what you mean here. But I'm willing to clean up in another PR.

Reply:
I mean, using a top level entry MuiList/MuiMenuItem in this components object, so that the styles would apply on all the lists or menu items, not just the ones inside a MuiPaper, as this seems to be a useless constraint.